go.mod

```diff
@@ -39,6 +39,7 @@ require (
 	github.com/emicklei/go-restful v2.9.5+incompatible
 	github.com/evanphx/json-patch v4.12.0+incompatible
 	github.com/fsnotify/fsnotify v1.4.9
+	github.com/go-openapi/spec v0.20.4
```

Comment:
this is spurious... `make clean generated_files` should drop the need for this

Reply:
Done

Reply:
still seeing other openapi changes in the go.mod files ... they should be dropped from the PR

Reply:
clean dropped it from go.su, but not go.mod. Removed it manually and confirmed the steps didn't re-add it. Should be good now.